src/Symfony/Bridge/Doctrine/Messenger/MessageRecordingEntityTrait.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Messenger;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ * @author Matthias Noback <matthiasnoback@gmail.com>
+ * @author Valentin Udaltsov <udaltsov.valentin@gmail.com>
+ */
+trait MessageRecordingEntityTrait
+{
+    /**
+     * @var object[]
```

Comment:
If this should go in 4.4 as a new feature, object typehint is not possible. Because 7.1.3 is ne minimum PHP version.

Reply:
It's too late for `4.4`, isn't it? That's why I targeted `master`, which will be `5.1`, I guess

Reply:
Not sure 😊